Stripe/STPPaymentMethod+Private.h

```diff
@@ -0,0 +1,20 @@
+//
+//  STPPaymentMethod+Private.h
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 3/12/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import "STPPaymentMethod.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface STPPaymentMethod ()
```

Comment:
nit: I find it useful to always give a name to a category, so something like `@interface STPPaymentMethod (Private)`

Reply:
Last week I learned that we usually *don't* want names here. Without a name it's a class extension, but adding a name turns it into a category.

The compiler will require all methods declared in the extension to have implementations. That restriction is relaxed for categories. It looks like the Faux Pas rule for that is disabled for stripe-ios, but I think it's still a good idea to prefer class extensions.

http://fauxpasapp.com/rules/#rule-PrivateCategory